docs/dev/table/connect.md

```diff
@@ -276,6 +276,40 @@ tables:
         type: VARCHAR
 {% endhighlight %}
 </div>
+
+<div data-lang="DDL" markdown="1">
+{% highlight sql %}
+-- CREATE a 010 version Kafka table start from the earliest offset(as table source) and append mode(as table sink).
+CREATE TABLE MyUserTable (
+  `user` bigint,
+  message varchar,
+  ts varchar
```

Comment:
Use `STRING` instead. We should not encourage users to use plain `VARCHAR`

Reply:
Firstly i use string but Jark said that the `STRING` is not usable now, i kindly agree.